### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader: NVDA announcing as expanded for Clustered Index Seek node.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/azuredatastudio#20764]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG4.1.2; #FTP; #NVDA; #Win11

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that NVDA announcing as expanded for Clustered Index Seek node.

### Actual:
NVDA announcing as expanded for Clustered Index Seek node. For Clustered Index Seek node sub-nodes are not presents still NVDA announcing as expanded.

**NOTE**: Issue is repro with JAWS

### Expected Result:
NVDA  should not announce as expanded for Clustered Index Seek node.

### Attachment:
![A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader (2)](https://user-images.githubusercontent.com/97668775/194806321-0567fd9b-5f2e-4841-bb9d-53498c4e39ac.gif)

**Comment by aasimkhan30** [comment]
@poojapatil36, this is fixed in the latest insiders. Can you please review it?

**Comment by poojapatil36** [comment]
Verified the issue in latest insiders. Issue is fixed with both NVDA and JAWS.
Please find below screenshot for reference.
![20764_Closing attachment](https://user-images.githubusercontent.com/97668775/197966831-9b7305ca-351d-4746-8abe-1dee246a4ccf.png)
![20764_Fixed](https://user-images.githubusercontent.com/97668775/197966837-616970c4-b808-42ec-8206-11c1029b205a.png)

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
